python/data_extraction/export_data.py

```diff
@@ -1,12 +1,16 @@
-import data
 from data_extraction import content_export
 from data_extraction import taxonomy_query
 from lib import plek
 from lib.helpers import dig
+from multiprocessing import Pool
```

Comment:
This doesn't seem to be used in this commit. If it is used later, maybe move it there?

Reply:
The diff makes it hard to see that I reordered / separated from ... import vs. import.